packages/components/src/Card/Card.tsx

```diff
@@ -4,23 +4,49 @@ import { XOR } from "ts-xor";
 import styles from "./Card.css";
 import colors from "./colors.css";
 import { CardClickable } from "./CardClickable";
+import { Button, ButtonProps } from "../Button";
 import { Heading } from "../Heading";
 
-interface CardProps {
+export type HeaderProps = XOR<HeaderActionProps, HeaderCustomProps>;
+interface CardCommonProps {
   /**
    * The `accent`, if provided, will effect the color accent at the top of
    * the card.
    */
   readonly accent?: keyof typeof colors;
   readonly children: ReactNode | ReactNode[];
+}
+
+interface CardHeaderProps {
+  readonly title?: never;
+
+  /**
+   * The header props of the card.
+   */
+  readonly header?: HeaderProps;
+}
 
+interface CardTitleProps {
   /**
-   * The title of the card.
+   * @deprecated Use header instead
    */
   readonly title?: string;
+
+  readonly header?: never;
```

Comment:
Similar I don't believe this is adding anything to our type system. Please correct me if I am wrong though.

Reply:
![image](https://user-images.githubusercontent.com/94410044/217956825-0cebfb23-7f98-40b6-8231-ccc318e381de.png)

Reply:
Sorry this comment was not about the `deprecated`  tag it was about defining `header: never`